src/chime_dash/__init__.py

```diff
@@ -8,32 +8,33 @@
 from typing import TypeVar
 from chime_dash.app.config import from_object
 from penn_chime.settings import get_defaults
-from chime_dash.app.components import Body
+from chime_dash.app.pages.root import Root
 from chime_dash.app.utils.callbacks import wrap_callbacks
 
 DashAppInstance = TypeVar('DashAppInstance')
 DEFAULTS = get_defaults()
```

Comment:
Not touched by this PR, but:

How often is the app initialized? Is this per request or per session?

How do I ensure that DEFAULTS updates to use the current date? This was the motivation for switching to a function call instead of a module global variable. It is being saved here as a module level global variable.